### PR TITLE
Parsing error sheet name fixed

### PR DESCRIPTION
**Pull request by omaus**
This PR
- fixes the errors #35 & #36 by replacing all `Option.defaultValue`s with `match ... with`s
- expands the sheet names the Investigation parser recognizes

### CODE REVIEW

**Comment by kMutagene** [comment]
can you please add a test where this is covered?

File: tests/ARCTokenization.Tests/TestUtils.fs [hunk 1]
```diff
@@ -55,12 +57,17 @@ module CvParam =
...
+    let structuralEquality (cvpExpected : CvParam) (cvpActual : CvParam) =
+        termNamesEqual cvpExpected cvpActual
+        accessionsEqual cvpExpected cvpActual
+        refUrisEqual cvpExpected cvpActual
+        valuesEqual cvpExpected cvpActual
+
+    let termEquality onto (cvpExpected : CvParam) (cvpActual : CvParam) =
+        let cvtExp = CvTerm.create(cvpExpected.Accession, cvpExpected.Name, cvpExpected.RefUri)
+        let cvtAct = CvTerm.create(cvpActual.Accession, cvpActual.Name, cvpActual.RefUri)
+        let synos
```
Comment (kMutagene) [inline tests/ARCTokenization.Tests/TestUtils.fs:69]: seems like something was left out unintentionally here

**Review by kMutagene — CHANGES_REQUESTED** [review]

File: tests/ARCTokenization.Tests/TestUtils.fs (hunk 1 above)
Comment (omaus) [reply tests/ARCTokenization.Tests/TestUtils.fs:69]: Yeah, it's still WIP. Changing it to draft therefore...

**Comment by kMutagene** [comment]
fixed via direct commit https://github.com/nfdi4plants/ARCTokenization/commit/560d46bfb11df5587f06a6da990bf49dad88334b,
thanks for investigating this. If you want to add a test covering this in the future, feel free to open another PR

**Pull request closed without merging**